recipes/packages.rb

```diff
@@ -34,11 +34,12 @@
          end
 
 packages = []
-packages += node['osl-php']['packages']
+packages += node['osl-php']['packages'].flatten
 
 # Prepend PHP package prefix to short packages (e.g. "php71u-memcached")
 if node['osl-php']['php_packages'].any?
-  packages += node['osl-php']['php_packages'].map { |pkg| prefix + '-' + pkg }
+  puts node['osl-php']['php_packages'].to_s
```

Comment:
I'm guessing this was an addition when debugging?

If we do want to output this for some reason, it should probably be logged properly.

Reply:
ooops yeah. I also noticed some imperfections in the README that I still need to address :(

Reply:
Removed now